### DISCUSSION TITLE
Welcome to Namida Discussions!

### DISCUSSION THREAD

**Discussion by MSOB7YY** [namidaco/namida#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our cute users. Here you can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Write opinions/reviews about Namida and how to improve it.
  Remember that we're all friends !

**Comment by idontveapc** [comment]
why yet no video player?

**Reply by MSOB7YY** [reply]
im still working on this, i have explained before in https://github.com/namidaco/namida/issues/40#issuecomment-1783106499, for reference im gonna quote

> 
> * currently videos are automatically played when
> 
> 1. track has a youtube link
> 2. there is a "linked" local video to the music
> 
> > refer to [video-integration](https://github.com/namidaco/namida#video-integration) for more info
> 
> * i should focus on bringing full video support soon, it was initially delayed to after few stuff be ready, like:
>   
>   * [x]  video player screen (and related gestures)
>   * [x]  general queue system
>   * [x]  general history system
>   * [x]  general playlist system
>   * [x]  heirarchy view for files
>   * [x]  video tags (information) extractor (and editor if needed)
> * now that all these stuff are ready, only thing left is a UI adaptation, shouldn't be so hard but gonna require some time, gonna work on it soon isa

- current plan is: 
   1. implement some more YouTube-related stuff (#1), like queue view and channel subscriptions
   2. complete local video player (this)

**Reply by idontveapc** [reply]
😁 :trollface:

**Comment by idontveapc** [comment]
are you an m or an f?

**Reply by MSOB7YY** [reply]
is this a proper question 💀 check "developer gender" thread in discord server for more info

**Reply by idontveapc** [reply]
okay, 😁

**Comment by danielb2** [comment]
thanks for adding this 😌

**Comment by UmarSidiki** [comment]
Can you please add Equalizer / Amplifier?

**Reply by MSOB7YY** [reply]
already requested, yeah but later

**Reply by MSOB7YY** [reply]
@UmarSidiki reminder its now available with v2.0.1

**Comment by Prokilo1987** [comment]
Hello namida team,

I want to say that your app is amazing and has real potential, this mail is just a few ideas to improve the app:


1 - synchronise app with Musicbrainz (its open source as well as their thumbnail database) this way the app can download thumbnails and rectify tags just like Power amp does, could be even better if you can use multiple data bases (spotify / apple music / sound cloud / music brainz / google) 


2 - add possibility to drag the scrolling cursor and slide it up and down when navigating library


3 - when listing by title add alphabetic characters on the side so we can go directly to the wanted character


4 - add possibility to add favorites to the library tabs (directly not just in playlists menu) 


5 - add possibility to navigate library by just seeing thumbnails where they would be 3 or 5 thumbnails in each line (3 rows x Nb of lines) 


I want to say that your app made me no longer a Poweramp user, I'm very inspired by your work and hope to see your app reach its true potential, I hope my ideas will be taken seriously, especially the thumbnails idea.

**Reply by MSOB7YY** [reply]
thank you sososos much, lets keep it at #239

**Reply by Prokilo1987** [reply]
So is that a yes?! 😅❤️
Will you try to add the download thumbnails feature?

**Reply by MSOB7YY** [reply]
lmao we discuss on the issue

**Comment by Prokilo1987** [comment]
Hello namida team,
I want to say that your app is amazing and has real potential, this mail is just a few ideas to improve the app:

1 - synchronise app with Musicbrainz (its open source as well as their thumbnail database) this way the app can download thumbnails and rectify tags just like Power amp does, could be even better if you can use multiple data bases (spotify / apple music / sound cloud / music brainz / google) 

2 - add possibility to drag the scrolling cursor and slide it up and down when navigating library

3 - when listing by title add alphabetic characters on the side so we can go directly to the wanted character

4 - add possibility to add favorites to the library tabs (directly not just in playlists menu) 

5 - add possibility to navigate library by just seeing thumbnails where they would be 3 or 5 thumbnails in each line (3 rows x Nb of lines) 

I want to say that your app made me no longer a Poweramp user, I'm very inspired by your work and hope to see your app reach its true potential, I hope my ideas will be taken seriously, especially the thumbnails idea. 


Also if you could add a way for us to do login that way the app saves our preferences, that would be great

Afficher le texte des messages précédents

**Comment by AbuBakar2006** [comment]
Will you make YouTube account login

**Reply by MSOB7YY** [reply]
working on it #227, btw its done in beta but still stuff to do. currently user feed/notifications/search is done (and user playlists in progress)

**Comment by nafizimtiaz005** [comment]
Why it is not working from a few days??

**Reply by MSOB7YY** [reply]
update to 4.8.6
